tc-tests-utils.sh

```diff
@@ -226,7 +251,19 @@ run_tflite_basic_inference_tests()
   assert_correct_ldc93s1 "${phrase_pbmodel_nolm}"
 }
 
-run_all_inference_tests()
+run_netframework_inference_tests()
```

Comment:
The output contains much more informations, so we just check that what we want is here.